Coffee3.4/src/CoffeeBrush.java

```diff
@@ -0,0 +1,206 @@
+import java.util.Scanner; //import Scanner from library
+public class CoffeeBrush {
+
+    private enum State {
+        Ready, SelectCoffee, Off, Filling
+    }
+
+    public Scanner scan = new Scanner(System.in);
+
+    /**
+     * buy options
+     * */
+    private final int espresso = 1;
+    private final int latte = 2;
+    private final int cappuccino = 3;
+
+    /**
+     * initial resources
+     * */
+    private int water = 400;
+    private int milk = 540;
+    private int coffee = 120;
+    private int cups = 9;
+    private int money = 550;
+
+    private State state = State.Ready;
+
+    CoffeeBrush(){
+        setReady();
+    }
+
+    /**
+     * one cup of espresso contains:
+     * */
+    private int e_water = 250;
+    private int e_coffee = 16;
+    private int e_cost = 4;
+
+    /**
+     * one cup of latte contains:
+     * */
+    private int l_water = 350;
```

Comment:
@aniterstepanyan use "camelCase" for fields name
https://www.geeksforgeeks.org/java-naming-conventions/#:~:text=Java%20uses%20CamelCase%20as%20a,letter%2C%20rest%20all%20with%20capital.

Reply:
Done